pallets/attestation/src/lib.rs

```diff
@@ -315,30 +338,20 @@ pub mod pallet {
 			max_parent_checks: u32,
 		) -> DispatchResultWithPostInfo {
 			let source = <T as Config>::EnsureOrigin::ensure_origin(origin)?;
-			let revoker = source.subject();
+			let attester = source.subject();
```

Comment:
S: At this point, we know that the subject is trying to revoke. We don't know yet if they are the attesters. I would personally change it back to revoker or find something else different than attester.

Reply:
The origin is often called `who` so I renamed it to `who` 50c9545eb11b170217b14c3ba1c3dd4ee8c7a9d5